packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -621,7 +622,30 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         containerScope: FluidObject = context.scope,
         existing?: boolean,
     ): Promise<ContainerRuntime> {
-        // If taggedLogger exists, use it. Otherwise, wrap the vanilla logger:
+        let existingFlag = true;
+        if (!existing) {
+            existingFlag = false;
+        }
```

Comment:
Please use the existing (no pun intended) logic, to be safe:

`const loadExisting = existing === true || context.existing === true`

(from L670 on the LHS)

This is especially important for properly staging the removal of context.existing.  It's not gone til it's gone!